remote-api/src/main/java/org/alfresco/repo/web/scripts/content/ContentStreamer.java

```diff
@@ -496,6 +498,20 @@ public void setAttachment(WebScriptRequest req, WebScriptResponse res, boolean a
             res.setHeader("Content-Disposition", headerValue);
         }
     }
+
+    private String encodeFilename(String attachFileName)
+    {
+        try
+        {
+            return Rfc5987Util.encode(attachFileName);
+        } catch (UnsupportedEncodingException e)

```

Comment:
The catch statement should be on its own line. This Alfresco style link is old but still applies [https://hub.alfresco.com/t5/alfresco-content-services-hub/coding-standards-for-alfresco-content-services/ba-p/290457](https://hub.alfresco.com/t5/alfresco-content-services-hub/coding-standards-for-alfresco-content-services/ba-p/290457)

Reply:
Ah, didn't notice, thanks.